online-inference/fastertransformer/gptj-ft-inference-service.yml

```diff
@@ -0,0 +1,40 @@
+apiVersion: serving.kubeflow.org/v1beta1
+kind: InferenceService
+metadata:
+  labels:
+    qos.coreweave.cloud/latency: low
+  name: fastertransformer-triton
+spec:
+  predictor:c
+    maxReplicas: 1
+    minReplicas: 1
+    containerConcurrency: 1
+    containers:
+    - name: gptj-6b-1-gpu-rtx-a5000
+      image: rtalaricw/gptj_ft:22.05
+      command: ["/opt/tritonserver/bin/tritonserver"]
+      args: ["--model-repository=/mnt/pvc/triton-model-store"]
+      env:
+        - name: STORAGE_URI 
+          value: pvc://model-storage/
+      ports:
+        - containerPort: 8000
+          protocol: TCP
+      resources:
+        requests:
+          cpu: 16
```

Comment:
Does fastertransformer actually use 16 CPUs per GPU? What does CPU usage actually peak out at?

Reply:
Changed to 4 CPUs and 8 Gi Memory as discussed.